ocs_ci/helpers/dr_helpers.py

```diff
@@ -1715,3 +1715,49 @@ def verify_drpc_deletion(cmd, expected_output_lst):
         if expected_output not in drpc_out.stderr.decode():
             return False
     return True
+
+
+def verify_last_kubeobject_protection_time(drpc_obj, kubeobject_sync_interval):
```

Comment:
can we rename this function as ```get_valid_last_kubeobject_protection_time``` because you are not only verifying it but also returning.

Reply:
created a issue https://github.com/red-hat-storage/ocs-ci/issues/11641